components/_global-footer.scss

```diff
@@ -88,11 +88,28 @@
 	}
 
 	&__bottom-bar-row {
+		margin: 0 auto;
 		padding: 16px 0;
 
 		&.wds-has-border-top {
 			border-top: 1px solid $color-slate-gray;
 		}
+
+		&.wds-has-no-padding {
```

Comment:
My intuition told me that the default row should be padded. The row with just a button seemed like an custom exception (yeah, I know there are only two rows 😝 ). And the option you're proposing might look like that:
```
	&__bottom-bar-row {
		margin: 0 auto;
		…
		&.wds-has-padding {
			padding: 16px 0;
		}
	}
```
…and it actually looks pretty damn good, now when I look at it. 👍